fcc-test-suite-template-master/index.html

```diff
@@ -40,6 +46,42 @@ <h1>Handcrafted, home-made masterpieces</h1>
           </ul>
         </form>
       </section>
+      <section id="features">
+        <div class="row">
+          <div class="col icon">
+            <i class="fa fa-3x fa-fire"></i>
+          </div>
+          <div class="col">
+            <h2>Premium Materials</h2>
+            <p>
+              Our trombones use the shiniest brass which is sourced locally.
+              This will increase the longevity of your purchase.
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col icon"><i class="fa fa-3x fa-truck"></i></div>
+          <div class="col">
+            <h2>Fast Shipping</h2>
+            <p>
+              We make sure you recieve your trombone as soon as we have finished
+              making it. We also provide free returns if you are not satisfied.
+            </p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col icon">
+            <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+          </div>
+          <div class="col">
+            <h2>Quality Assurance</h2>
+            <p>
+              For every purchase you make, we will ensure there are no damages
```

Comment:
For every purchase you make, we will ensure that there are no damages
              or faults and will also check and test the pitch of your instrument